### ISSUE TITLE
Funannotate update with pre-assembled transcriptome and pasa gff and pasadb

### ISSUE THREAD

**Issue by JoelShore** [nextgenusfs/funannotate#428]
Hello
Sorry to bother you.
I wish to use funannotate update.
I had previously used trinity to assemble transcriptome.
I have run funannotate predict and have the gff3 from it too.
I have also run pasa and I have the gff3 file from it, and the sqlite database it produced during the
run. I hope to just feed the assembled transcriptome in (i.e. not start back at fastq reads)
the pasa gff, the pasadb and the gff3 file (or gbk file) from the previous funannotate run.
It isn't clear to me that this will run, as it seems that update needs to begin with fastq reads?

Note that I can't run trinity on my current machine as I don't have sufficient RAM
for it (I'm running with 8 cores and 32 gig of ram).
My main purposes for running update is to annotate UTRs onto the genome.

Thanks
Joel

Ok continuing .. I am now running funannotate update and it appears to be running and using the sqlite database I previously prepared.
However, update requires reads, so I gave it my assembled transcriptome in fasta format but claimed they were nanopore reads. Note this was the same transcriptome I gave it to generate the pasa database initially and that I used in funannotate predict. Not sure this is legitimate and it seems circular.
In any case, it seems to be running and is now at step one of pasa comparison and is writing stuff to the sqlite database.

I'm not sure the results - if it runs successfully, will be meaningful however.

Ok this has indeed run.
The run took less than 24 hours although it is not complete and still parsing
the final gkb file. It has written the final gff3 file which does indeed contain
5 and 3 prime UTRs.

I'd greatly appreciate any thoughts on this approach.
Thanks
Joel

**Comment by JoelShore** [comment]
Sorry that should say HELLO!

**Comment by JoelShore** [comment]
Hello
Sorry to bother you.
I wish to use funannotate update.
I had previously used trinity to assemble transcriptome.
I have run funannotate predict and have the gff3 from it too.
I have also run pasa and I have the gff3 file from it, and the sqlite database it produced during the
run. I hope to just feed the assembled transcriptome in (i.e. not start back at fastq reads)
the pasa gff, the pasadb and the gff3 file (or gbk file) from the previous funannotate run.
It isn't clear to me that this will run, as it seems that update needs to begin with fastq reads?

Note that I can't run trinity on my current machine as I don't have sufficient RAM
for it (I'm running with 8 cores and 32 gig of ram).
Thanks
Joel

**Comment by JoelShore** [comment]
Ok continuing .. I am now running funannotate update and it appears to be running and using the sqlite database I previously prepared.
However, update requires reads, so I gave it my assembled transcriptome in fasta format but claimed they were nanopore reads. Note this was the same transcriptome I gave it to generate the pasa database initially and that I used in funannotate predict. Not sure this is legitimate and it seems circular.
In any case, it seems to be running and is now at step one of pasa comparison and is writing stuff to the sqlite database.

I'm not sure there results - if it runs successfully, will be meaningful however.

I'd greatly appreciate any thoughts on this.
Thanks
Joel
